cmd/validator/validator.go

```diff
@@ -181,6 +191,19 @@ Supported formats: standard, json, junit (default: "standard")`,
 	return config, nil
 }
 
+func buildExcludeFileTypesFromFileTypes(excludeFileTypesPtr, fileTypesPtr *string) error {
+	if excludeFileTypesPtr != nil && fileTypesPtr != nil && *excludeFileTypesPtr != "" && *fileTypesPtr != "" {
+		return errors.New("Cannot use --exclude-file-types and --file-types together")
```

Comment:
I wouldn't use the leading hyphens here

These options are also controlled by the environment variables, no?